Doc/library/functions.rst

```diff
@@ -1165,7 +1165,11 @@ are always available.  They are listed here in alphabetical order.
    *buffering* is an optional integer used to set the buffering policy.  Pass 0
    to switch buffering off (only allowed in binary mode), 1 to select line
    buffering (only usable in text mode), and an integer > 1 to indicate the size
-   in bytes of a fixed-size chunk buffer.  When no *buffering* argument is
+   in bytes of a fixed-size chunk buffer. Note that specifying a buffer size this
+   way does not apply for files opened in text mode, where
```

Comment:
"does not apply" is misleading because it is applied to underlying buffered random.

How about: "applies for binary buffered I/O, but TextIOWrapper would have another buffering."

Reply:
Hmm, for the latter part would it be possible to say instead "..., but TextIOWrapper (i.e., files opened with `mode='r+'`) would have another buffering"? I feel that the issue was brought up as the buffering parameter is misleading for text mode, so saying `TextIOWrapper` alone might not be explicit enough

Reply:
It looks good to me.

Reply:
'It' referring to the original suggested change or the one with `mode`?